IntroToCSharp/Shared/User/UserData.cs

```diff
@@ -29,15 +29,19 @@ public class UserStats
     /// <value>Amount of gold</value>
     public int Gold { get; private set; }
 
+    public int GoldSpent {get; set;} //When the store 
+    public int GoldAcquired {get; private set;}
     /// <summary>
     /// Constructs the user's xp and gold.
     /// </summary>
     /// <param name="xp">The amopunt of xp.</param>
     /// <param name="gold">The amount of gold.</param>
-    public UserStats(int xp, int gold)
+    public UserStats(int xp, int gold, int goldAcquired, int goldSpent)
```

Comment:
I think we should be able to remove `gold` and make `Gold` a property that is calculated based on `GoldAcquired` and `GoldSpent`